Game/Src/World/world.gd

```diff
@@ -16,56 +16,57 @@ onready var swordL2 = $TestEnemyMovement/Character/sword/Sprite_L2
 onready var TestEnemyMouvement = $TestEnemyMovement
 
 func _ready():
-#	connect('potion',self,"increase_health",[-0.05])
+	$CanvasLayer/GloryUI.set_glory(GlobalVariables.glory)
 	pass
 
 func _process(_delta):
 
 	if GlobalVariables.glory > 0.000001 and flag == false:
```

Comment:
If you have an idea of what "flag" is, can rename it?
also I would delete instead of comment the set_glory call. We can always revert to the previous state
(Same for all the comments in this file actually)
-edit- It looks like you actually deleted it in the next commit. I leave the comment in case it is not deleted but you can probably ignore it :)

Reply:
No idea for flag.
Indeed, I'll delete everything, it was just for ease of correction if I made a mistake